newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -59,6 +64,16 @@ export default class InstancesSelection {
       },
       'ns-resize'
     );
+    this._makeButton(
+      this.rotateButton,
+      event => {
+        this.onRotate(event.deltaX, event.deltaY);
+      },
+      () => {
+        this.onRotateEnd();
+      },
+      'url("res/actions/rotate24.png"),auto'
```

Comment:
Nice! I was wondering how you managed to get that rotation cursor :D

Reply:
thank you 😄  unfortunately css cursors dont have an icon for rotation, so I had to use one of gdevelop's instead